state/machine.go

```diff
@@ -178,12 +178,10 @@ func (m *Machine) ContainerType() instance.ContainerType {
 	return instance.ContainerType(m.doc.ContainerType)
 }
 
-func (m *Machine) ModelName() string {
```

Comment:
The only use of this was in a test. Replaced and removed to discourage potato programming.